creusot/tests/should_succeed/iterators/07_fuse.rs

```diff
@@ -73,8 +77,8 @@ impl<I: Iterator> Invariant for Fuse<I> {
     #[predicate]
     fn invariant(self) -> bool {
```

Comment:
This shouldn't be required but we still need it in this case due to the lack of specialization